language/German/strings.po

```diff
@@ -1664,179 +1664,179 @@ msgstr "Menü zur Softwareauswahl überspringen"
 msgid "Hide romless machine from available list"
-msgstr ""
+msgstr "Maschine ohne ROMs"
 
 #: src/frontend/mame/ui/submenu.cpp:39
 msgid "Advanced Options"
-msgstr ""
+msgstr "Erweiterte Optionen"
 
 #: src/frontend/mame/ui/submenu.cpp:40
 msgid "Performance Options"
-msgstr ""
+msgstr "Leistungs-Optionen"
 
 #: src/frontend/mame/ui/submenu.cpp:41
 msgid "Auto frame skip"
-msgstr ""
+msgstr "Bilder automatisch auslassen"
 
 #: src/frontend/mame/ui/submenu.cpp:42
 msgid "Frame skip"
-msgstr ""
+msgstr "Bilder auslassen"
 
 #: src/frontend/mame/ui/submenu.cpp:43
 msgid "Throttle"
-msgstr ""
+msgstr "Begrenzung"
 
 #: src/frontend/mame/ui/submenu.cpp:44
 msgid "Sleep"
-msgstr ""
+msgstr "Schlafe"
 
 #: src/frontend/mame/ui/submenu.cpp:45
 msgid "Speed"
-msgstr ""
+msgstr "Geschwindigkeit"
 
 #: src/frontend/mame/ui/submenu.cpp:46
 msgid "Refresh speed"
-msgstr ""
+msgstr "Auffrisch-Geschwindigkeit"
 
 #: src/frontend/mame/ui/submenu.cpp:48
 msgid "Rotation Options"
-msgstr ""
+msgstr "Rotations-Optionen"
 
 #: src/frontend/mame/ui/submenu.cpp:49 src/frontend/mame/ui/videoopt.cpp:204
 msgid "Rotate"
 msgstr "Drehen"
 
 #: src/frontend/mame/ui/submenu.cpp:50
 msgid "Rotate right"
-msgstr ""
+msgstr "Nach recht rotieren"
 
 #: src/frontend/mame/ui/submenu.cpp:51
 msgid "Rotate left"
-msgstr ""
+msgstr "Nach links rotieren"
 
 #: src/frontend/mame/ui/submenu.cpp:52
 msgid "Auto rotate right"
-msgstr ""
+msgstr "Automatisch nach rechts rotieren"
 
 #: src/frontend/mame/ui/submenu.cpp:53
 msgid "Auto rotate left"
-msgstr ""
+msgstr "Automatisch nach links rotieren"
 
 #: src/frontend/mame/ui/submenu.cpp:54
 msgid "Flip X"
-msgstr ""
+msgstr "X umkehren"
 
 #: src/frontend/mame/ui/submenu.cpp:55
 msgid "Flip Y"
-msgstr ""
+msgstr "Y umkehren"
 
 #: src/frontend/mame/ui/submenu.cpp:57
 msgid "Artwork Options"
-msgstr ""
+msgstr "Illustrations-Optionen"
 
 #: src/frontend/mame/ui/submenu.cpp:58
 msgid "Artwork Crop"
-msgstr ""
+msgstr "Illustrationen beschneiden"
 
 #: src/frontend/mame/ui/submenu.cpp:59
 msgid "Use Backdrops"
-msgstr ""
+msgstr "Hintergründe verwenden"
 
 #: src/frontend/mame/ui/submenu.cpp:60
 msgid "Use Overlays"
-msgstr ""
+msgstr "Verwende Overlays"
 
 #: src/frontend/mame/ui/submenu.cpp:61
 msgid "Use Bezels"
-msgstr ""
+msgstr "Verwende Rahmen"
 
 #: src/frontend/mame/ui/submenu.cpp:62
 msgid "Use Control Panels"
-msgstr ""
+msgstr "Verwende Steuerungselemente"
 
 #: src/frontend/mame/ui/submenu.cpp:63
 msgid "Use Marquees"
-msgstr ""
+msgstr "Verwende Laufschriften"
 
 #: src/frontend/mame/ui/submenu.cpp:65
 msgid "State/Playback Options"
-msgstr ""
+msgstr "Status/Wiedergabe-Optionen"
 
 #: src/frontend/mame/ui/submenu.cpp:66
 msgid "Automatic save/restore"
-msgstr ""
+msgstr "Automatisch speichern/wiederherstellen"
 
 #: src/frontend/mame/ui/submenu.cpp:67
 msgid "Bilinear snapshot"
-msgstr ""
+msgstr "Bilineare Bildschirmfotos"
 
 #: src/frontend/mame/ui/submenu.cpp:68
 msgid "Burn-in"
-msgstr ""
+msgstr "Einbrennen"
```

Comment:
"Bildschirm eingebrannt" finde ich auch in ordnung und Einbrennung/en gibt es tatsächlich nicht ;) .